apple/Sources/FerrostarMapLibreUI/Extensions/ManeuverImageKeys.swift

```diff
@@ -0,0 +1,66 @@
+import FerrostarCoreFFI
+import Foundation
+
+// TODO: See if this could probably be in the core reasonably?
```

Comment:
Do we want to move this to common?

Reply:
Discussed on the call: we can leave this as-is for now. Probably will have a different approach for Android anyways.